eventmon/monitord/impl.go

```diff
@@ -221,6 +232,35 @@ func receiveV0(reader io.Reader) (eventmon.EventV0, error) {
 }
 
 func (m *Monitor) writeHtml(writer io.Writer) {
+	fmt.Fprintln(writer, "Keymasters:<br>")
+	fmt.Fprintln(writer, `<table border="1">`)
+	fmt.Fprintln(writer, "  <tr>")
+	fmt.Fprintln(writer, "    <th>IP Addr</th>")
+	fmt.Fprintln(writer, "    <th>Status</th>")
+	fmt.Fprintln(writer, "  </tr>")
+	m.mutex.RLock()
```

Comment:
Since you are using defers why not defer here after the lock?

Reply:
Yes. That's what I intended. I inserted the new code prior to the defer :-/